tests/vmcheck/test-kernel-args.sh

```diff
@@ -180,6 +180,22 @@ vm_rpmostree kargs --delete-if-present=PACKAGE3=TEST3 --unchanged-exit-77 || rc=
 assert_streq $rc 77
 echo "ok exit 77 when unchanged kargs with unchanged-exit-77"
 
+# Test for 'rpm-ostree kargs --editor'.
+vm_rpmostree kargs > kargs.txt
+assert_not_file_has_content_literal kargs.txt 'nonexisting'
+rc=0
+EDITOR="sed -i 's/nonexisting//'" vm_rpmostree kargs --editor --unchanged-exit-77 || rc=?
```

Comment:
Does this actually work?  I don't think we forward `EDITOR` across `ssh`, or do we?

Reply:
He added a hunk to forward it in `vm_rpmostree`.

Reply:
I'm bruteforcing this iteratively. Things I've unearthed so far:
 * need to forward `EDITOR` explicitly
 * need to take care of the common case where `EDITOR` is unset/unbound
 * need to tell ssh to keep command line intact with `--`